### DISCUSSION TITLE
convert marimo to typical python as download jupyter notebook

### DISCUSSION THREAD

**Discussion by NguyenVanThanhHust** [marimo-team/marimo#796] [General]
I usually use jupyter notebook to dev and test new class. After that, I would download my_class.ipynb as my_class.py then import from other file to use.

Is there anyway to do that in marimo?

**Comment by akshayka** [comment]
Hi! There isn't a way to do that yet.

Very soon, we will add the ability to import classes you developed in marimo, without having to change the file format.

Something like:

```python
import my_marimo_notebook.app as notebook

MyClass = notebook.classes.MyClass
```

In the future we may also add a utility to convert a marimo notebook to a Python script that does not depend on marimo, as you suggest.
